app/views/users/_users.html.erb

```diff
@@ -0,0 +1,20 @@
+
+<div class="init-list">
+  <% users.each do |user| %>
+    <%= link_to edit_user_path(user), :style=> "text-decoration: none;" do %>
+      <div class="card">
+        <h3 class="init-fullname"><%= user.fullname %></h3>
+          <p class="init-email"><%= user.email %></p>
+            <% if user.is_admin == true %>
```

Comment:
This section of logic could use the same treatment as we have in index. I'm sorry for not noticing it sooner.

Reply:
Altered logic to represent same treatment in index. No worries!